frontend/src/components/Explanation.js

```diff
@@ -1 +1,15 @@
-// This is where the explanation component will be created
\ No newline at end of file
+import '../styles/Explanation.css';
+import React from 'react';
+
+function Explanation()
+{
+    return(
+        <div class="jumbotron">
+            <h1 class="h1Size">Welcome to PetFinder!</h1>
```

Comment:
Is the styling for the h1Size class just for h1 in general. If so, it might be better to remove the class and just style it in the App css file as h1{} so that this styling occurs for all h1 elements.

Reply:
I agree with Rosie here as well.

Reply:
No, this styling is just applied for the header on the explanation. I don't think it should apply for all headers on the website.